go.mod

```diff
@@ -21,7 +21,7 @@ require (
 
 require (
 	github.com/diggerhq/lib-digger-config v0.1.13
-	github.com/diggerhq/lib-orchestrator v0.0.11
+	github.com/diggerhq/lib-orchestrator v0.0.12-0.20231003151127-0bb18578b47b
```

Comment:
it looks like it's using commit and not some specific release ?

Reply:
yeah I will update properly once the other PR is merged